pkg/minikube/kubeconfig/config.go

```diff
@@ -178,3 +182,58 @@ func decode(data []byte) (*api.Config, error) {
 
 	return config.(*api.Config), nil
 }
+
+// GetKubeConfigStatus verifys the ip stored in kubeconfig.
+// If incorrect, the ip is overwritten.
+func GetKubeConfigStatus(ip string, filename string) (string, error) {
+	if ip == "" {
+		return "", fmt.Errorf("Error, empty ip passed")
+	}
+	ks := ""
+	kip := ""
+	con, err := ReadConfigOrNew(filename)
+	if err != nil {
+		return "", errors.Wrap(err, "Error getting kubeconfig status")
+	}
+	if con.Clusters["minikube"] == nil {
```

Comment:
I think an `cluster, ok := ` check might be better than nil here. Then you can reuse that value below in the con.Clusters["minikube"] line.